app/models/concerns/sequenced.rb

```diff
@@ -16,6 +18,12 @@ def ensure_sequential_id
       self.sequential_id = generate_sequential_id
     end
 
+    def ensure_organization_sequential_id
+      return if organization_sequential_id.present? && organization_sequential_id > 0
```

Comment:
Style/NumericPredicate: Use `organization_sequential_id.positive?` instead of `organization_sequential_id > 0`. (https://rubystyle.guide#predicate-methods)